Jenkinsfile

```diff
@@ -1057,7 +1060,8 @@ pipeline {
                     steps {
                         functionalTest inst_repos: el7_daos_repos(),
                                        inst_rpms: get_daos_packages('centos7') +
-                                                  ' ' + functional_rpms
+                                                  ' ' + functional_rpms +
+                                                  ' python2-tabulate '
```

Comment:
Ditto.

Reply:
[python2-tabulate](https://centos.pkgs.org/6/epel-i386/python2-tabulate-0.7.7-1.el6.noarch.rpm.html) is now in [daos.spec](https://github.com/daos-stack/daos/blob/master/utils/rpms/daos.spec). See 60c8ef19a03e1a7e5aaee2a59e7c2ab60f6d9362.